.github/dependabot.yml

```diff
@@ -35,3 +35,23 @@ updates:
       - codingllama
       - rosstimothy
       - zmb3
+
+  - package-ecosystem: cargo
+    directory: "/"
+    schedule:
+      interval: weekly
+    open-pull-requests-limit: 10
+    reviewers:
+      - codingllama
+      - rosstimothy
+      - zmb3
+
+  - package-ecosystem: cargo
```

Comment:
Is this necessary? We use cargo workspaces, so rdpclient is "linked" from the root Cargo.toml. It might work without this block.

Reply:
I'm not 100% sure. I tried doing `cargo update` from the root as a way to check it and it doesn't seem to have an effect, whereas `cargo update` inside the `rdpclient` folder does update the dependencies. Based on that, it looks to me that the `/` path is actually the one we don't need (but I played it safe and kept it too).